### PR TITLE
Close & continue

### PR DESCRIPTION
**Pull request by thecalcc**

### CODE REVIEW

**Comment by thecalcc** [comment]
Add button

**Comment by tomaskikutis** [comment]
> Add button in react authoring
> Handle isDirty from angular in React

do these and move out the logic from sdApi to authoringApiCommon

**Comment by thecalcc** [comment]
After merging https://github.com/superdesk/superdesk-client-core/pull/4231 pull latest changes from post-broadcasting and reuse the new logic for `C & C`. Implement a new function to handle C & C conditional display.

File: scripts/api/article.ts [hunk 1]
```diff
@@ -383,6 +389,10 @@ function isEditable(_article: IArticle): boolean {
     }
 }
 
+function rewrite(item: IArticle) {
```
Comment (tomaskikutis) [inline scripts/api/article.ts:392]: can you add a return type?

**Review by tomaskikutis — APPROVED** [review]

**Pull request merged**